portals/application/configs/klear/model/Users.yaml

```diff
@@ -180,22 +180,19 @@ production:
       trim: both
       maxLength: 65
     maxCalls:
-      title: _('Max call')
+      title: _('Max calls')
       type: number
-      defaultValue: 2
+      defaultValue: 1
```

Comment:
This is not the default value specified in the delta below (where database defaults to 0)